common/src/primitives/compact.rs

```diff
@@ -136,19 +136,4 @@ mod tests {
         err_conversion(0x01fedcba);
         err_conversion(!0x00800000); // overflow
     }
-    /*
-
-    #[test]
-    fn test_min_difficulty() {
```

Comment:
Can we reenable this?

Reply:
I think the test is not correct: it assumes that roundtrip conversion is possible between Compact <-> UInt256.

Reply:
I think it should be possible for minimum difficulty.

Reply:
hmmm, not sure the test doesn't pass and I thought it should:
```
Uint256::MAX => Compact(0x2100FFFF)
Compact(0x2100FFFF) => Uint256(0xFFFF0000, 0x0, 0x0, 0x0)
```